include/ebpf_result.h

```diff
@@ -91,6 +91,9 @@ extern "C"
 
         /// The enumeration found no more keys.
         EBPF_NO_MORE_KEYS, // = 25
+
+        // The requested key is already present.
+        EBPF_KEY_ALREADY_EXISTS, // = 26
```

Comment:
```suggestion
        EBPF_KEY_ALREADY_EXISTS,
```
I think the convention (from Anurag?) is that only every 5th one is annotated with the number